hello_phoenix/README.md

```diff
@@ -36,7 +36,9 @@ see the [`vintage_net` documentation](https://hexdocs.pm/vintage_net/).
     ```bash
     # Still in ui directory from the prior step.
     # These steps need to be repeated when you change JS or CSS files.
+    cd assets
     node node_modules/webpack/bin/webpack.js --mode production
+    cd ..
```

Comment:
Can we actually just change this whole section to be remove the need to change directories?

```suggestion
    npm install --prefix assets --production
```

Reply:
@jjcarstens -- definitely, you can remove the lines but you lose the benefits of webpack.  Personally, I liked a simple approach, especially for example projects.  There is no need to complicate things by putting in webpack.  I like your suggestion better.